albatross.ml

```diff
@@ -606,11 +644,15 @@ module Make (S : Tcpip.Stack.V4V6) = struct
                 m "init: raw_query failed for %s: %s"
                   (Configuration.name_to_str configuration.name)
                   str);
+            let error =
+              Status.make `Configuration
+                (Fmt.str "Initialisation error: raw_qeury failed for %s with %s"
+                   (Configuration.name_to_str configuration.name)
+                   str)
+            in
             let state =
               {
-                error =
-                  Some
-                    (Fmt.str "Failed to query this instance with error: %s" str);
+                status = Offline [ error ];
```

Comment:
Do we have recovery from `Offline` (is there a button to try communicating with this albatross)?